assets/xml/objects/object_mori_tex.xml

```diff
@@ -1,5 +1,30 @@
 <Root>
-    <File Name="object_mori_tex" Segment="6">
-        <Blob Name="object_moriTex_Blob_000000" Size="0x9E00" Offset="0x0"/>
+    <File Name="object_mori_tex" Segment="8">
+        <Texture Name="gMoriElevatorBarTex" OutName="elevator_bar" Format="rgba16" Height="16" Width="64" Offset="0x0"/>
+        <Texture Name="gMoriElevatorTopTex" OutName="elevator_top" Format="rgba16" Height="32" Width="32" Offset="0x800"/>
+        <Texture Name="gMoriElevatorSideTex" OutName="elevator_side" Format="rgba16" Height="16" Width="16" Offset="0x1000"/>
+
+        <Texture Name="gMorieStoneWallTex" OutName="stone_wall" Format="rgba16" Height="32" Width="32" Offset="0x1200"/>
+        <Texture Name="gMoriKaitenkabeMetalWallTex" OutName="metal_wall" Format="rgba16" Height="32" Width="32" Offset="0x1A00"/>
+        <Texture Name="gMoriRakkatenjoCheckerboardTex" OutName="checkerboard" Format="rgba16" Height="32" Width="64" Offset="0x2200"/>
+
+        <Texture Name="gMoriStalfosPlatformSideTex" OutName="stalfos_platform_side" Format="rgba16" Height="32" Width="64" Offset="0x3200"/>
+        <Texture Name="gMoriStalfosPlatformTopTex" OutName="stalfos_platform_top" Format="rgba16" Height="32" Width="32" Offset="0x4200"/>
+
+        <Texture Name="gMoriHashiraPlatformsTex" OutName="hashira_platform" Format="rgba16" Height="32" Width="32" Offset="0x4A00"/>
+
+        <!-- This is just a guess since its unused -->
+        <Texture Name="gMoriWaterTex" OutName="water_tex" Format="rgba16" Height="8" Width="32" Offset="0x5200"/> 
+
+        <Texture Name="gMoriHashigoLadderTex" OutName="ladder" Format="rgba16" Height="32" Width="32" Offset="0x5400"/>
+
+        <Texture Name="gForestTwistedHallwayPillarTex" OutName="twisted_hall_pillar" Format="rgba16" Height="64" Width="16" Offset="0x5C00"/>
+        <Texture Name="gForestTwistedHallwayWallAndFloorTex" OutName="twisted_hall_wall_and_floor" Format="rgba16" Width="32" Height="32" Offset="0x6400"/>
```

Comment:
```suggestion
        <!-- This is just a guess since its unused -->
        <Texture Name="gMoriWaterTex" OutName="mori_water" Format="rgba16" Height="8" Width="32" Offset="0x5200"/> 

        <Texture Name="gMoriHashigoLadderTex" OutName="mori_hashigo_ladder" Format="rgba16" Height="32" Width="32" Offset="0x5400"/>

        <Texture Name="gForestTwistedHallwayPillarTex" OutName="forest_twisted_hall_pillar" Format="rgba16" Height="64" Width="16" Offset="0x5C00"/>
        <Texture Name="gForestTwistedHallwayWallAndFloorTex" OutName="forest_twisted_hall_wall_and_floor" Format="rgba16" Width="32" Height="32" Offset="0x6400"/>
```
Maybe there was a discussion about styling a while back, and maybe there's a good motivation for not having prefixes. But I find having the prefix to these texture names helps finding them a lot easier. Especially for generic textures such as `ladder` where there might be a lot of ladders to search through. thoughts?

Reply:
My thinking was that if you wanted to modify this texture you would go to the folder first then just find ladder. 
And I remember when we first did objects we started with no prefix cause they were the equivalent of private members since they are in a folder.

Reply:
yea we typically dont worry about prefixes in file names, only symbol names

Reply:
I don't think there was a ruling on this in the past, there probably should be one eventually as the repo is currently inconsistent about it

My personal opinion is the context of the object name is enough as the folder name, and filenames don't need to double down with the object name as a prefix

(alternatively I would also vote to get rid of OutName and just use the symbol name for the pngs too, in the name of being able to find what textures (pngs) are for which symbol, but last I asked that's not popular)

(on this object xml specifically though, `water_tex` probably shouldn't have `tex`)

Reply:
Thanks for explaining! +1 on `water_tex` -> `water`, last thing from me